src/objective-c/GRPCClient/private/GRPCWrappedCall.m

```diff
@@ -112,7 +112,7 @@ - (instancetype)initWithMessage:(NSData *)message handler:(void (^)())handler {
 }
 
 - (void)dealloc {
-  gpr_free(_op.data.send_message);
+  grpc_slice_buffer_destroy(_op.data.send_message);
```

Comment:
`_op.data.send_message` needs to be a `grpc_byte_buffer`. Looking at [its definition](https://github.com/grpc/grpc/blob/master/include/grpc/impl/codegen/grpc_types.h#L55), the slice buffer is encapsulated within it. So I assume [`grpc_byte_buffer_destroy`](https://github.com/grpc/grpc/blob/master/include/grpc/byte_buffer.h#L70) will take care of it, and is the right choice here.